app/Server.php

```diff
@@ -14,6 +14,31 @@ class Server extends Model
         'status'     => 'boolean',
     ];
 
+    /**
+     * Find server with the lowest usage
+     * @return Server|null
+     */
+    public static function lowestUsage()
+    {
+        $servers = self::where('status', true)->where('offline', false)->get();
+        $sorted  = $servers->sortBy(function ($server, $key) {
```

Comment:
This could also be done with a mysql query!

Reply:
Maybe the `usage` should be defined as an acessor?

Reply:
I changed it to an acessor and an sql query